velox/exec/benchmarks/OrderByBenchmark.cpp

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <folly/Benchmark.h>
+#include <folly/init/Init.h>
+
+#include "glog/logging.h"
+#include "velox/exec/tests/utils/AssertQueryBuilder.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::exec;
+
+namespace {
+
+struct TestCase {
+  explicit TestCase(
+      const std::string& testName,
+      size_t numRows,
+      int numVectors,
+      const RowTypePtr& rowType,
+      int numKeys)
+      : testName{testName},
+        numRows{numRows},
+        numVectors{numVectors},
+        rowType{rowType},
+        numKeys{numKeys} {}
+
+  std::string testName;
+  size_t numRows;
+  int numVectors;
+  RowTypePtr rowType;
+  int numKeys;
+};
+
+class OrderByBenchmark {
```

Comment:
It appears that this benchmark and PrefixSortBenchmark share some code. I wonder if we can extract the common implementations to a separate file like SortBenchmarkUtil.cpp and reuse these methods. Thanks.